system/env.go

```diff
@@ -22,11 +22,12 @@ import (
 )
 
 const (
-	NamespaceEnvKey = "SYSTEM_NAMESPACE"
+	NamespaceEnvKey   = "SYSTEM_NAMESPACE"
+	ConfigLabelEnvKey = "SYSTEM_CONFIG_LABEL"
```

Comment:
The label that comes to mind here (which we used in the config validation webhook) is `foo.knative.dev/release`.  I think that we should probably call this the `SYSTEM_RESOURCE_LABEL` (even though both current uses are configmaps, we could have more in the future), and use it both places.

Reply:
That's exactly the one I was thinking about. I can rename it, that makes sense.